lib/code_corps/github/sync/sync.ex

```diff
@@ -26,7 +26,7 @@ defmodule CodeCorps.GitHub.Sync do
                  | {:error, :validating_github_issue}
                  | {:error, :validating_github_comment}
                  | {:error, :validating_user}
-                 | {:error, :validating_tasks}
+                 | {:error, :validating_task}
                  | {:error, :validating_comments}
```

Comment:
We're gonna be `:validating_comment` with this to, or am I mistaken?

Reply:
I'm not quite positive. Part of why I need a thorough poring over this.